smartsheet-csharp-sdk/main/Smartsheet/Api/Internal/SheetCrossSheetReferencesResourcesImpl.cs

```diff
@@ -26,8 +26,15 @@ namespace Smartsheet.Api.Internal
     using Smartsheet.Api.Models;
     using Smartsheet.Api.Internal.Util;
 
+    /// <summary>
+    /// Cross sheet references impl class for resources
```

Comment:
nitpick: seems like in the comments/documentation implementation could be written out for clarity (all places) but you could convince me either way

Reply:
I think they should all be fixed now with my latest commit.

Reply:
thanks!